### PR TITLE
Update rubygem-katello to 3.18.1.1 for internal testing

### PR DESCRIPTION
**Pull request by ianballou**
We are bumping rubygem-katello to 3.18.1.1 so we can test the Pulp 3 migration reset, which is merging straight into KATELLO-3.18 rather than master.

The only change in 3.18.1.1 is the Pulp 2 to 3 migration bindings update to 0.7.  There is also a PR to bump the bindings version for rpm/2.3: https://github.com/theforeman/foreman-packaging/pull/6320.

Since it's for testing, we aren't making a public release out of it. This is likely to happen again since the master branch has diverged so much from 3.18 Pulp3-wise.

### CODE REVIEW

**Comment by ianballou** [comment]
The errors are due to the missing `tfm-rubygem(pulp_2to3_migration_client) >= 0.7.0`

**Comment by jlsherrill** [comment]
[test katello]

**Comment by jlsherrill** [comment]
[test rpm]

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -12,7 +12,7 @@
 Name:    %{?scl_prefix}rubygem-%{gem_name}
 Summary: Content and Subscription Management plugin for Foreman
 
-Version: %{mainver}
+Version: 3.18.1.1
```
Comment (jlsherrill) [inline packages/katello/rubygem-katello/rubygem-katello.spec:15]: a script may have done this, but we usually update the mainver variable above

**Review by jlsherrill — APPROVED** [review]

**Pull request merged**